src/accountsdb/db.zig

```diff
@@ -1858,7 +1904,16 @@ pub const AccountsDB = struct {
             self.account_index.reference_allocator,
             n_accounts,
         );
-        try self.account_index.validateAccountFile(account_file, bin_counts, &references);
+
+        try indexAndValidateAccountFile(
+            account_file,
+            self.account_index.pubkey_bin_calculator,
+            bin_counts,
+            &references,
+            // NOTE: this method should only be called in tests/benchmarks so we dont need
```

Comment:
Could you name the method which is only called in tests/benchmarks? Or perhaps add doc comment on the method which should only be called for tests/benchmarks

Reply:
f367f54